sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedForm.cs

```diff
@@ -22,13 +22,16 @@ internal ExtractedForm(DocumentResult_internal documentResult, ICollection<PageR
         {
             // Supervised
             LearnedFormType = documentResult.DocType;
-            PageRange = new PageRange(documentResult.PageRange);
+            StartPageNumber = documentResult.PageRange.First();
```

Comment:
Can PageRange be empty? `First` and `Last` would throw funny exceptions in that case.

Reply:
This will go away in the next design iteration. https://github.com/Azure/azure-sdk-for-net/issues/10547